backend/api.py

```diff
@@ -9,6 +9,10 @@
 api = Flask(__name__)
 api.debug = False
 
+database = Database()
```

Comment:
Database's constructor does logging, so this must happen after logging is configured.

Reply:
@AndreasWintherMoen Changed the order of execution so the DB is not instantiated before the logger is configured.